x/data/server/features/attest.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Attest
+
+  Scenario: data is attested to when the data has been anchored
+    Given a valid content hash
+    And alice has anchored the data at block time "2020-01-01"
+    When alice attempts to attest to the data at block time "2020-01-02"
+    Then an error of ""
+    And the data id entry exists
+    And the data anchor entry exists and the timestamp is equal to "2020-01-01"
+    And the data attestor entry exists and the timestamp is equal to "2020-01-02"
```

Comment:
we should be testing usually one thing per scenario, unless it's a "journey" scenario. this seems like too many ands

Reply:
Updated to test one thing per scenario (i.e. separate tests for anchor entry and attestor entry)